### DISCUSSION TITLE
When would there be a new release?

### DISCUSSION THREAD

**Discussion by wgundamj44** [deeplook/svglib#396] [General]
the latest version 1.5.1 was published a year ago, would there be a new release to pick up the changes within last year?

**Comment by deeplook** [comment]
Very true. Thanks for the reminder. I think we should get the builds on GitHub working again which stopped building for reportlab 4.x due to some cairo issues. I should have a related email by @replabrobin and find/undust it in my inbox. Will put it here, shortly, i.e. earlier than usual. Sorry for the delays!

**Reply by deeplook** [reply]
See below. It takes some testing and minor additions to `.github/workflows/ci.yml`. I'm moving it up on my todo list. 

> How about making the bitmap renderer a choice to be made by the user, choosing between library, pycairo or anything else? Then we could stick with librart for CI/CD and recommend pycairo for production use.

The rl-renderPM accelerator is now available in pypi as rl_renderPM see https://pypi.org/project/rl-renderPM/
to force its use you can export the environment variable RL_renderPMBackend=_renderPM.
